coderd/prometheusmetrics/prometheusmetrics.go

```diff
@@ -200,7 +200,7 @@ func Agents(ctx context.Context, logger slog.Logger, registerer prometheus.Regis
 			})
 			if err != nil {
 				logger.Error(ctx, "can't get workspace rows", slog.Error(err))
-				continue
+				goto done
```

Comment:
Not a blocker, but prefer to avoid goto when possible. Introduces a more complex flow.

In this instance e.g. an if/else might be preferable even though it introduces indentation.

Reply:
Fixed, although I prefer "goto" in this situation :)